### PR TITLE
Improve SmartSeq2 portability

### PR DESCRIPTION
**Pull request by mckinsel**
This PR has a few of changes to the SS2 test and pipeline that help it run better in other environments.

### CODE REVIEW

File: library/tasks/RSEM.wdl [hunk 1]
```diff
@@ -29,7 +29,7 @@ task RSEMExpression {
   command {
     set -e
   
-    tar -xvf ${rsem_genome}
+    tar --no-same-owner -xvf ${rsem_genome}
```
Comment (jishuxu) [inline library/tasks/RSEM.wdl]: `rsem` and `hisat2` indexing files are `tar.gz`. is it okay to just use `tar -xvf` to extract `.tar.gz` file?

Comment (rexwangcc) [reply library/tasks/RSEM.wdl]: Good point here! If they are gzip files, `-z` will be necessary here I guess

Comment (mckinsel) [reply library/tasks/RSEM.wdl]: You actually don't need the `-z` for gnu tar. See here: https://www.gnu.org/software/tar/manual/tar.html#SEC135

"Reading compressed archive is even simpler: you don't need to specify any additional options as GNU tar recognizes its format automatically."

Also, I think the PR tests would have failed.

Comment (rexwangcc) [reply library/tasks/RSEM.wdl]: I see, thanks!

Comment (jishuxu) [reply library/tasks/RSEM.wdl]: good to know this, thanks!

File: library/tasks/HISAT2.wdl [hunk 2]
```diff
@@ -8,7 +8,7 @@ task HISAT2PairedEnd {
 
   # runtime values
   String docker = "quay.io/humancellatlas/secondary-analysis-hisat2:v0.2.2-2-2.1.0"
-  Int machine_mem_mb = 5000
+  Int machine_mem_mb = 15000
```
Comment (jishuxu) [inline library/tasks/HISAT2.wdl]: why do you request `15G` ?

Comment (mckinsel) [reply library/tasks/HISAT2.wdl]: These tasks request 4 cores and 5GB of memory. On GCP, this should get you an n1-standard-4, which actually has 15GB of memory. On AWS, you might get a c4.xlarge, which has 7.5GB of memory.

Some of the tasks appear to actually require between 7.5 and 15GB of memory, so they succeed on GCP but fail on AWS even though both platforms provide an instance that meets the declared requirements.

Comment (jishuxu) [reply library/tasks/HISAT2.wdl]: Yes, it is fair. But 15G is actually overkill for this task. On average, HISAT2 uses ~8G so I would like to request 10G instead 15G here. Does `10G` work with AWS?

Comment (mckinsel) [reply library/tasks/HISAT2.wdl]: It does, but just like GCP you'll get an instance with 15 or 16GB of memory. I'm a little reluctant to set it lower than what we're actually using in production. If 8GB is average but say 10% of jobs use >10GB, then we'd never notice problems in the Green Box, but other users would see lots of failures.

**Pull request merged**